mrp_production_category_notes/models/product_category.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025 Angel Rivas <angel.rivas@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    mrp_notes = fields.Html(
+        string="Manufacturing order notes",
+        help="Add notes to manufacturing orders",
+        translate=True,
+    )
+
+    def get_mrp_notes(self):
+        self.ensure_one()
+        mrp_notes = ""
+
+        if self.mrp_notes and self.mrp_notes not in ["<p><br></p>", ""]:
```

Comment:
You can simply do this:

`if self.mrp_notes and self.mrp_notes != "<p><br></p>":`

If mrp_notes was "", the first coundition (self.mrp_notes) would return False already.